nsinit/README.md

```diff
@@ -5,13 +5,15 @@ It is able to spawn new containers or join existing containers.
 
 ### How to build?
 
-First to add the `libcontainer/vendor` into your GOPATH. It's because something related with this [issue](https://github.com/docker/libcontainer/issues/210).
+First to add the `libcontainer/vendor` into your GOPATH. It's because something
+related with this [issue](https://github.com/docker/libcontainer/issues/210).
```

Comment:
It is because libcontainer vendors all its dependencies, so it can be built predictably.